secant/Util/Strings.swift

```diff
@@ -8,3 +8,56 @@ extension String {
         self.init(stringLiteral: output)
     }
 }
+
+extension String {
+    private static let floatingPointRegex = "^[0-9]*.?[0-9]+"
```

Comment:
This regex might be an example, but it's wrong. In many languages the dot is the group separator, not the decimal separator.

Reply:
I just put a placeholder in here, it would obviously be trivial to update these regexes as you need

Reply:
sounds fair. Let's add an issue referencing that an a todo then so we don't miss it 👍